odoo/addons/base/models/res_company.py

```diff
@@ -223,7 +224,8 @@ def install_l10n_modules(self):
         uninstalled_modules = self.uninstalled_l10n_module_ids
         is_ready_and_not_test = (
             not tools.config['test_enable']
```

Comment:
How come `tools.config['test_enable']` is not set? :thinking:

Reply:
Why would it be? You can call install_demo and a standard configuration